react-app/src/App.tsx

```diff
@@ -12,6 +13,7 @@ const router = createBrowserRouter([
     children: [
       { index: true, element: <HomePage /> },
       { path: "TablePage", element: <TablePage /> },
+      { path: "GridPage", element: <GridPage /> },
```

Comment:
let's change path to grid, urls should be lowercase and "Page" is redundant as it's a subpage